src/app/components/shared/menu/widgets/allows-original-download/allows-original-download.component.ts

```diff
@@ -56,10 +48,12 @@ export class AllowsOriginalDownloadComponent
       : `Owner of ${name} has not set any permissions for allowing the downloads of the audio recordings yet`;
   }
 
-  private determineIfUserHasAccess(project: Project): boolean {
+  public getUserAccess(project: Project): string {
     const { allowOriginalDownload, accessLevel } = project;
-    return allowOriginalDownload
-      ? hasRequiredAccessLevelOrHigher(allowOriginalDownload, accessLevel)
-      : false;
+
+    return allowOriginalDownload &&
+      hasRequiredAccessLevelOrHigher(allowOriginalDownload, accessLevel)
+      ? "Allowed"
+      : "Not Allowed";
```

Comment:
the short answer here: if they have write access to a project they can add annotations

we should add a capability for this